2021-01-21 20:58:19,115 INFO    MainThread:220048 [wandb_setup.py:_flush():70] setting env: {}
2021-01-21 20:58:19,115 INFO    MainThread:220048 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-01-21 20:58:19,115 INFO    MainThread:220048 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-21 20:58:19,115 INFO    MainThread:220048 [wandb_init.py:_log_setup():318] Logging user logs to K:\OneDrive - Cumberland Valley School District\Education\Activates\Science Fair\PCG-Science-Fair\PCG-arrhythmia-detection\wandb\run-20210121_205819-26cs09df\logs\debug.log
2021-01-21 20:58:19,115 INFO    MainThread:220048 [wandb_init.py:_log_setup():319] Logging internal logs to K:\OneDrive - Cumberland Valley School District\Education\Activates\Science Fair\PCG-Science-Fair\PCG-arrhythmia-detection\wandb\run-20210121_205819-26cs09df\logs\debug-internal.log
2021-01-21 20:58:22,480 INFO    MainThread:220048 [wandb_run.py:_console_start():1308] atexit reg
2021-01-21 20:58:22,482 INFO    MainThread:220048 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.WRAP
2021-01-21 20:58:22,482 INFO    MainThread:220048 [wandb_run.py:_redirect():1206] Wrapping output streams.
2021-01-21 20:58:22,482 INFO    MainThread:220048 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-21 20:58:22,482 INFO    MainThread:220048 [wandb_watch.py:watch():39] Watching
2021-01-21 20:58:25,124 INFO    MainThread:220048 [wandb_run.py:_config_callback():608] config_cb None None {}
2021-01-21 20:58:29,063 INFO    MainThread:220048 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-21 20:58:29,064 INFO    MainThread:220048 [wandb_run.py:_restore():1250] restore
2021-01-21 20:58:29,337 INFO    MainThread:220048 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "479f04a1bbf04e3b9e5fa7facc8e6a31"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 878
      total_bytes: 878
    }
  }
}

2021-01-21 20:58:31,346 INFO    MainThread:220048 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "fb0c6fd56cf94a1f85a22a59d4b50ddd"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 8219
      total_bytes: 8219
    }
  }
}

2021-01-21 20:58:32,472 INFO    MainThread:220048 [wandb_run.py:_show_files():1614] logging synced files
2021-01-21 20:58:32,481 INFO    MainThread:220048 [internal.py:handle_exit():139] Internal process exited
